packages/@okta/vuepress-site/docs/concepts/saml/index.md

```diff
@@ -2,36 +2,36 @@
+Most apps have a user store (database or LDAP) that contains, among other things, user profile information and credentials. When a user signs in, the credentials are validated against this user store. The advantage of this approach is that everything is managed within the app, providing a single and consistent way to authenticate a user.
 
-However, if a user needs to access multiple applications where each one requires a different set of credentials, it becomes a problem for the end user. First, the user needs to remember different passwords, in addition to any other corporate password (for example, their AD password) that may exist. The user is now forced to maintain separate usernames and passwords, and must handle different password policies and expirations. In addition, this scenario also creates a headache for administrators and ISVs when application users continue to have access to applications that should have been revoked.
+However, if a user needs to access multiple apps where each one requires a different set of credentials, it get complicated. First, the user needs to remember different passwords in addition to any other corporate password (for example, their AD password) that may exist. The user is now forced to maintain separate usernames and passwords, and must handle different password policies and expirations. This scenario also creates headaches for admins and independent software vendors (ISV) when users continue to have access to apps that should have been revoked.
 
-## Federated Identity
+## Federated identity
 
-Federated Identity started with the need to support application access that spans beyond a company or organization boundary. Imagine a relationship between a juice company (JuiceCo) selling its product to a large supermarket chain (BigMart). As an employee of JuiceCo, you need to access an application provided by BigMart to manage the relationship and monitor supplies and sales.
+Federated identity from a need to support app access beyond a company or organization boundary. Imagine a relationship between a juice company (JuiceCo) selling its product to a large supermarket chain (BigMart). As an employee of JuiceCo, you need to access an app provided by BigMart to manage the relationship and monitor supplies and sales.
 
-In this case, BigMart (who is providing this application) will need to take care of user authentication. The simple way is to require a different username and password from users working at JuiceCo. But think about all the users that this application need to maintain - including all other suppliers and their users who need to access the application.
+In this case, BigMart (who is providing this app) needs to handle user authentication. The simple way is to require a different username and password from users working at JuiceCo. But with many users to maintain, it isn't that simple, especially with all the suppliers and their users who need to access the app.
```

Comment:
```suggestion
In this case, BigMart (who is providing this app) needs to manage user authentication. The simple way is to require a different username and password from users working at JuiceCo. But with many users to maintain, this can be complicated - especially with all the suppliers and their users who need to access the app.
```